manoseimas/mps_v2/views.py

```diff
@@ -108,6 +108,8 @@ def mp_fraction(request, fraction_slug):
     return render(request, 'fraction.jade', context)
 
 
+
```

Comment:
This is whitespace police says you have too many blank lines between functions (should be 2).